Add recipe for org-re-reveal

lechten opened the pull request:
### Brief summary of what the package does

This package provides a maintained fork of org-reveal, which is no longer maintained. As such, it provides Org export functionality to generate HTML presentations with the presentation framework reveal.js.

See PR #5979 for background and naming, which I'll close in a minute.

### Direct link to the package repository

https://gitlab.com/oer/org-re-reveal

### Your association with the package

Maintainer

### Relevant communications with the upstream package maintainer

Currently not possible.

### Checklist

Please confirm with `x`:

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses).
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

purcell commented:
Thanks! I think that a couple more of the `reveal` references should be changed to `re-reveal`: the custom group name and the backend name, specifically. (It's fine to leave the various `:reveal-*` options as-is for compatibility, of course, but the goal is to able to have both `org-reveal` and `org-re-reveal` installed and for them to not interfere with each other.) Looks good otherwise.

lechten commented:
I changed that. I also changed the export key bindings to lower case to have different ones from `org-reveal`. Besides, I added `htmlize` as dependency as some Org versions (e.g., 9.1.9 shipping with Emacs master) throw an error when exporting `Readme.org` otherwise.

purcell commented:
Great, thanks - merged now. :-)

lechten commented:
Many thanks for your feedback and efforts, I really appreciate your amazing work!
